x-pack/plugins/stack_alerts/public/rule_types/es_query/util.ts

```diff
@@ -41,4 +42,21 @@ export const convertFieldSpecToFieldOption = (fieldSpec: FieldSpec[]): FieldOpti
     });
 };
 
+export const convertRawRuntimeFieldtoFieldOption = (
+  rawFields: estypes.MappingRuntimeFields
+): FieldOption[] => {
+  const result: FieldOption[] = [];
+  for (const name of Object.keys(rawFields)) {
+    const rawField = rawFields[name];
+    const type = rawField.type;
+
+    const normalizedType = type;
```

Comment:
The normalized type is not the same as the type for numerical values. Here is where it's mapped in the server: https://github.com/elastic/kibana/blob/4ed60697e97b7120eec7d0130da28ca900ca90e9/x-pack/plugins/triggers_actions_ui/server/data/routes/fields.ts#L132-L142

Maybe you could move that definition to the triggers_actions_ui common folder for reuse?

Reply:
Resolved in this commit [35dc771](https://github.com/elastic/kibana/pull/160319/commits/35dc77189a8e30056028b143c2d4d6bbfb17c0ff)